azure-pipelines.yml

```diff
@@ -1,120 +0,0 @@
-trigger:
-  branches:
-    include:
-    - master
-    - feature/ci-dev
-  paths:
-    exclude:
-    - README.md
-    - LICENSE
-    - .travis.yml
-    - book.toml
-    - docker/*
-    - manual/*
-    - docs/*
-
-jobs:
-- job: Linux
-  timeoutInMinutes: 120
-  pool:
-    vmImage: 'ubuntu-latest'
-  strategy:
-    matrix:
-      arch:
-        containerImage: immunant/c2rust:archlinux-base-latest
-      debian10:
-        containerImage: immunant/c2rust:debian-buster-latest
-      debian11:
-        containerImage: immunant/c2rust:debian-bullseye-latest
-      fedora34:
-        containerImage: immunant/c2rust:fedora-34-latest
-      ubuntu20:
-        containerImage: immunant/c2rust:ubuntu-focal-latest
-      ubuntu18:
-        containerImage: immunant/c2rust:ubuntu-bionic-latest
```

Comment:
Can we test on all of these distros and versions in GitHub Actions, too, instead of just Ubuntu 24?

Reply:
Oh wait, GitHub Actions only supports Ubuntu, right?  So we'll have to drop testing on all the others.  @thedataking is this okay?

Reply:
@kkysen It's theoretically possible to run a custom container of an arbitrary distro on GitHub Actions, but one of the huge benefits of this change to the CI is avoiding the separate (and manual) step of building containers, which makes it impossible for contributors to modify and test (because contributors can't replace those containers). With the new approach, all the packages are installed on the fly, and it's much easier to iterate on the result.

I think that usability improvement would be worth making this change.